.github/workflows/pull-request-workflow.yml

```
@@ -62,3 +62,33 @@ jobs:
         run: yarn install --immutable
       - name: Build
         run: yarn build
+
+  tests:
+    needs:
+      - build
+    runs-on: ubuntu-latest
+    container: node:16-alpine3.15
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_PASSWORD: example
+          POSTGRES_DB: test_integration
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+    steps:
+      - uses: actions/checkout@v3
```

If matrix works here, don't forget this one 

```suggestion
      - uses: actions/checkout@v3
         with:
           node-version: ${{ matrix.node-version }}
```

---

Ok

---

Okay, do we have another way how we can use matrix style for our integration tests?

---

Yes, done.